src/templates/about/index.hbs

```diff
@@ -47,9 +48,9 @@
         </div>
         <div class="text">
           <h2>Scott Noell</h2>
-          <p>With 30 years' experience in the financial planning industry, Scott Noell is a partner and key driver of the FinSec Partners business. Scott built his impressive career- foundations specifically within insurance and superannuation before applying
+          <p>With 30 years experience in the financial planning industry, Scott Noell is a partner and key driver of the FinSec Partners business. Scott built his impressive career foundations specifically within insurance and superannuation before applying
             himself to more holistic financial planning practices.</p>
-          <p>Over the last 15 years Scott has actively developed the pension transfer business within FinSec, originally in a working relationship with Pension Transfers Direct and now via FinSec PTX.</p>
+          <p>Over the last 15 years Scott has actively developed the pension transfer business within FinSec, originally in a working relationship with Pension Transfers Direct and now via FinSec’s specialist transfer service FinSec PTX.</p>
```

Comment:
In fact have a look in `src/templates/partials/bad-characters`. I've collected all the characters that shouldn't be in the code there.

Except for the &. That one can be both JS, CSS and HTML. However in HTML it performs a special function. So I sometimes look for all & in the HTML and look if the all part of a special HTML character. Basically look for "& " (so ampersand + space) because that should never occur in HTML.